lib/rules/extends-correct-class.js

```diff
@@ -31,29 +55,42 @@ module.exports = {
         const classRef = classes.get(node.arguments[1])
         if (!classRef) return
         const extendsTag = getExtendsOption(node.arguments[2])
-        const desiredSuperName = builtInTagMap[extendsTag] || 'HTMLElement'
+
+        // Allowed super names is an array of allowed names.
+        let allowedSuperNames
+        if (builtInTagMap[extendsTag]) {
+          allowedSuperNames = [builtInTagMap[extendsTag]]
+        } else {
+          allowedSuperNames = [...((context.options[0] || {}).allowedSuperNames || [])]
+          // Make sure to treat this like a set.
```

Comment:
If we'd like to treat this like a Set, does it make sense to just make it a Set?

Reply:
I think changing it to a `Set` also works!

To stringify with the changes, would we need to convert back to an array?

Is this worth the overhead, going from .eslintrc.json array -> Set object -> array? It would make lookups faster